### PR TITLE
Api provides journalists to tag article as premium

### PR DESCRIPTION
**Pull request by offcast-dev**
PT: https://www.pivotaltracker.com/story/show/178256265
### User Story
```
As a owner
To be able to monetize the app
I want some articles to be tagged as premium
```
### Changes Proposed
- adds premium attribute to Article
- refactors seed to include premium

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -51,6 +51,7 @@
     t.text "body"
     t.bigint "user_id"
```
Comment (arunbhalli) [inline db/schema.rb:52]: what is  bigint here can we not use int only?

File: db/seeds.rb [hunk 2]
```diff
@@ -48,8 +48,7 @@
                    first_name: 'Mr.', last_name: 'Fake', role: 5)
 
 (0...titles.count).each do |i|
- article = Article.create(title: titles[i], teaser: teasers[i], body: body[i], category: categories[rand(categories.count - 1)],
-                 user_id: user.id)
+ article = Article.create(title: titles[i], teaser: teasers[i], body: body[i], category: categories[rand(categories.count - 1)], user_id: user.id, premium: [true, false].sample)
```
Comment (christerforsgren91) [inline db/seeds.rb:51]: what is this .sample doing?

File: db/schema.rb (hunk 1 above)
Comment (offcast-dev) [reply db/schema.rb:52]: integer holds values up to 2147483647, and bigint up to 9223372036854775807. Probably we can work with integer but id attributes are created by rails automatically and it seem like bigint is the default setting

File: db/seeds.rb (hunk 2 above)
Comment (offcast-dev) [reply db/seeds.rb:51]: sample is the ruby method you can use on any array. It picks one random value from the array. In this case it randomly picks one of 'true' or 'false'. This way when we seed, premium attribute will be assigned randomly to different articles

Comment (christerforsgren91) [reply db/seeds.rb:51]: Allright 👍

**Review by christerforsgren91 — APPROVED** [review]

**Review by arunbhalli — APPROVED** [review]

**Review by LucasKnudsen — COMMENTED** [review]
Let's add premium to the serializers and maybe a lil test

**Review by LucasKnudsen — APPROVED** [review]

**Pull request merged**